.github/workflows/ci.yaml

```diff
@@ -166,25 +85,33 @@ jobs:
       run: |
         pdm run integration
 
+    - name: Docker Compose Down
```

Comment:
How does this call `down`? it looks identical to the `up` block

Reply:
Mmm.. good point. Too much cargoculting from GPT4. :P Lemme fix..

Reply:
https://github.com/recap-build/recap/pull/422 😅